scene/gui/base_button.cpp

```diff
@@ -535,6 +547,7 @@ void BaseButton::_bind_methods() {
 	ADD_SIGNAL(MethodInfo("toggled", PropertyInfo(Variant::BOOL, "button_pressed")));
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "disabled"), "set_disabled", "is_disabled");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "toggle_mode"), "set_toggle_mode", "is_toggle_mode");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "show_input_tooltip"), "set_show_input_tooltip", "is_show_input_tooltip_enabled");
```

Comment:
Maybe `tooltip_with_shortcut_info`?

Reply:
Or `add_shortcut_to_tooltip`? It's hard to find something not too long but still descriptive enough :D

Reply:
`shortcut_in_tooltip` maybe.

Reply:
`shortcut_in_tooltip` seems like a good option.